Bump security-scan image version in CIS-benchmark

doflamingo721 opened the pull request:
Linked issue:
https://github.com/rancher/cis-operator/issues/149

dhruvmewada15 reviewed (dismissed):
LGTM

prachidamle commented:
Need to run `make remove` to delete earlier rc version

doflamingo721 commented:
@prachidamle I've removed the older RC version. Can you please review it again? Thanks.

prachidamle commented:
@doflamingo721 did we run the `make remove` command - that would delete the rc1 and older assets. And we should add the new rc2 changes and assets and then run `make charts`

doflamingo721 commented:
@prachidamle Yes I had removed the rc1 assets and had added the rc2 changes, however I missed to update the version from rc1 to rc2. I have updated the versions now. Can you please review it again? Thanks.

prachidamle reviewed (dismissed):
LGTM

doflamingo721 commented on release.yaml:
```diff
@@ -23,5 +23,7 @@ rancher-csp-adapter:
+- 2.0.5-rc2
 rancher-cis-benchmark-crd:
 - 2.0.5-rc1
+- 2.0.5-rc2
```
I can see only one version in release.yaml.

prachidamle replied:
yeah, this was a comment I had posted earlier - not sure why it showed up now. I dont have this issue anymore.

macedogm commented on packages/rancher-cis-benchmark/charts/values.yaml:
```diff
@@ -8,7 +8,7 @@ image:
     tag: v1.0.8
```
@doflamingo721 can we also bump the `cis-operator` image to [v1.0.9](https://github.com/rancher/cis-operator/releases/tag/v1.0.9), please?
I did only an image refresh to retrieve updates and security patches. No additional modification was done.

macedogm reviewed (approved):
LGTM
